### PR TITLE
TATB- 29-Add header section for routing all page

### PR DESCRIPTION
**Pull request by PrabhatJTDeveloper**

### CODE REVIEW

File: src/app/component/header/header.component.html [hunk 1]
```diff
@@ -64,8 +46,8 @@
 </nav>
 
 <div
-  class="modal fade"
-  id="exampleModal"
+  class="modal"
```
Comment (charmijtdev) [inline src/app/component/header/header.component.html:49]: why model ? as already told that no need for model

File: src/app/component/header/header.component.html [hunk 2]
```diff
@@ -116,17 +100,13 @@ <h1>Jyoti Technosoft LLP</h1>
 
 <div
   class="modal"
```
Comment (charmijtdev) [inline src/app/component/header/header.component.html]: Please remove this model to and use model service for this

File: src/app/component/header/header.component.html [hunk 3]
```diff
@@ -139,25 +119,50 @@ <h1 class="modal-title fs-5">Alert</h1>
...
+          >
+            Cancel
+          </button>
+        </div>
       </div>
     </div>
   </div>
 </div>
+
+<!-- <div class="conatiner">
```
Comment (charmijtdev) [inline src/app/component/header/header.component.html]: remove this unwanted code

File: src/app/component/header/header.component.scss [hunk 4]
```diff
@@ -1,37 +1,24 @@
-*{
-    font-family: Roboto;
+* {
```
Comment (charmijtdev) [inline src/app/component/header/header.component.scss]: also please remove this class to

File: src/app/component/header/header.component.ts [hunk 5]
```diff
@@ -14,20 +14,18 @@ export class HeaderComponent {
   constructor(private route: Router, public questionService: QuestionService) {}
 
   ngOnInit() {
-    this.userName = this.questionService?.userName;
+    this.getData();
     if (!this.userName) {
```
Comment (charmijtdev) [inline src/app/component/header/header.component.ts]: remove this conidtion as not used anymore

**Review by charmijtdev — CHANGES_REQUESTED** [review]

File: src/app/component/header/header.component.html (hunk 3 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/header/header.component.html]: ok I have Remove code

File: src/app/component/header/header.component.scss (hunk 4 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/header/header.component.scss]: ok I have Remove class

File: src/app/component/header/header.component.html (hunk 1 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/header/header.component.html:49]: I have Remove model

File: src/app/component/header/header.component.html (hunk 2 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/header/header.component.html]: ok I have done Remove model

File: src/app/component/header/header.component.ts (hunk 5 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/header/header.component.ts]: ok I have done remove this condition

**Pull request merged**